smartsim/_core/_cli/utils.py

```diff
@@ -56,28 +57,50 @@ def color_bool(trigger: bool = True) -> str:
     return colorize(str(trigger), color=_color)
 
 
-def pip_install(packages: t.List[str], end_point: t.Optional[str] = None, verbose: bool = False) -> None:
+def pip_install(
+    packages: t.List[str], end_point: t.Optional[str] = None, verbose: bool = False
+) -> None:
     """Install a pip package to be used in the SmartSim build
     Currently only Torch shared libraries are re-used for the build
     """
     # form pip install command
-    cmd = ["python", "-m", "pip", "install"]
-    cmd.extend(packages)
+    cmd = [sys.executable, "-m", "pip", "install"]
```

Comment:
+1billion

Reply:
hahaha anything to try and be a little more safe!